app/src/main/res/layout/activity_main.xml

```diff
@@ -4,59 +4,163 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@drawable/background"
     tools:context=".MainActivity">
 
-    <EditText
-        android:id="@+id/etUsername"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="100dp"
-        android:ems="10"
-        android:hint="Username"
-        android:textStyle="bold|italic"
-        android:inputType="textPersonName"
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/etFirstName"
+        android:layout_width="322dp"
```

Comment:
As much as possible try not to give fixed sizes to views. Let's change the fixed width to `match_parent`

Reply:
alright sure